apps/melding-form/src/app/(general)/locatie/Location.tsx

```diff
@@ -22,7 +23,7 @@ import { postLocationForm } from './actions'
 
 import styles from './Location.module.css'
 
-const initialState: Pick<FormState, 'systemError' | 'validationErrors'> = {}
+const initialState: { systemError?: unknown; validationError?: ValidationError } = {}
```

Comment:
I'm not sure changing this to a singular error is wise. This deviates from the pattern we use on all other form pages and makes it easier for bugs to slip in. I suggest we keep it as it was.